src/plugins/intel_cpu/src/config.h

```diff
@@ -20,6 +20,12 @@
 namespace ov {
 namespace intel_cpu {
 
+enum class LatencyCandidateScope {
```

Comment:
1. Lets to declaration inside `Config` structure.
2. I would propose to rename -> `LatencyThreadingMode`
3. Is the same logic extension is expected for Throughput mode?

Reply:
Updated. 

Not yet. `LatencyThreadingMode` is defined for threads of one stream in Latency mode. We do not expect same configuration in both Latency mode and Throughput mode.